src/components/singleArticle.jsx

```diff
@@ -1,35 +1,55 @@
 import { useParams, Link } from "react-router-dom";
 import { useState, useEffect } from "react";
-import { getSingleArticle } from "../utils/api";
+import { getSingleArticle, updateVotes } from "../utils/api";
 
 
 const SingleArticle = () => {
 
     const [currentArticle, setCurrentArticle] = useState({})
+    const [articleVotes, setArticleVotes] = useState(0)
     const [isLoading, setIsLoading] = useState(true)
+    const [isError, setIsError] = useState(false)
     let {article_id} = useParams()
 
     useEffect(() => {
         getSingleArticle(article_id)
         .then((data) => {
             setCurrentArticle(data.article);
+            setArticleVotes(data.article.votes)
             setIsLoading(false);
         })
```

Comment:
I would consider using a catch block and the error state here as well just in case as we're looking to eventually handle this situation (article ID that doesn't exist being put into URL) in the future (ticket 12):

![image](https://user-images.githubusercontent.com/61753159/192513347-6a34dae0-22a8-4e5a-be4a-1b49a5004c1b.png)